cogs/utils/p_strings.py

```diff
@@ -159,17 +162,18 @@ def __init__(self,
         self.groups = groups
         self.additional_info = additional_info
 
+    @property
+    def patterns_and_values(self):
```

Comment:
i'd make this private (add a _ in front)

Reply:
Resolved in f780970